oudedetai.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Ou Dedetai
+GenericName=FaithLife Wine App Installer
+Comment=Installs and manages either Logos or Verbum via Wine and Flatpak. Community supported.
+Exec=/app/bin/entrypoint.sh
+Icon=/app/share/icons/hicolor/512x512/apps/oudedetai.png
```

Comment:
these should not be full paths.

Reply:
also icon name must match Flatpak ID and without the png extension.

Reply:
How can this desktop file be in upstream if the icon path uses the flatpak id?

Reply:
> if the icon path uses the flatpak id?

edit it with `desktop-file-edit` in the manifest.

Reply:
> How can this desktop file be in upstream if the icon path uses the flatpak id?

You can use the same convention upstream. Or can use the original **icon name** and use `rename-icon` in the manifest that will deal with it (that mean icons are also installed with their original name). As said never a path and never the file extension